packages/api-client/src/importData.ts

```diff
@@ -132,8 +132,14 @@ export const importData = async (
         tempData.map(async (item: any) => {
           item = await uploadFiles(item, tableSchema, filestackClient, fileUploadInfo.path);
 
-          const data = formatDataForMutation(MUTATION_TYPE.CREATE, tableName, item, schema, {
-            skip: (value: any, fieldSchema: FieldSchema) => tableFieldSelectors.isRelationField(fieldSchema),
+          const data = formatDataForMutation({
```

Comment:
@ej9x why do you change interface of this function? is it necessary? for what?

Reply:
@zouxuoz Because there will be appName argument. And function with 5 arguments is complicated to understand.

Reply:
@ej9x yes, but I think putting all arguments to the one object argument is overkill, maybe we could use 2 or 3 arguments?

@vorobeez what do u think about that?

Reply:
Maybe something like that? I think we should group only meta arguments.
```
formatDataForMutation(
  type: MUTATION_TYPE,
  data: any,
  meta: {
    tableName: string,
    appName: string,
    schema: Object,
  },
  options: Object,
)
```

Reply:
@vorobeez Why type is not part of meta?

Reply:
@vorobeez looks good

Reply:
fixed

Reply:
@ej9x I think mutation type is about output format, not about 8base schema and their path.